src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,79 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
+  display: block;
+  margin-top: 32px;
+  text-align: center;
+  width: 100%;
+}
+
+.w-subscribe__border {
+  border: 2px solid #f1f3f4;
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+  width: 100%;
+}
+
+.w-subscribe__button {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.w-subscribe__error {
+  color: $WARNING_COLOR;
+}
+
+.w-subscribe__field {
+  display: flex;
+  justify-content: space-between;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 8px;
+  text-align: left;
+}
+
+.w-subscribe__form {
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Can remove these if we decide to make the fields full bleed. Based on the design below I _think_ they're meant to be full width. We might have to fiddle with it a bit to see what we prefer.